atak-civ/plugin-examples/plugintemplate/app/src/main/java/com/atakmap/android/plugintemplate/TimerNotificationActionReceiver.java

```diff
@@ -0,0 +1,41 @@
+package com.atakmap.android.plugintemplate;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.widget.Toast;
+
+/**
+ * This class interfaces with notifications to process actions like pause or resume
+ */
+public class TimerNotificationActionReceiver extends BroadcastReceiver {
+    /**
+     * Override the onreceive function of broadcast receiver
+     * @param context application context
+     * @param intent passed in when creating the notification
+     */
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        int notifyId = intent.getIntExtra("activeTimer", -1);
+        //break if notifyId is messed up
+        if(notifyId == -1) {
+            return;
+        }
+        ActiveTimer cur = PluginTemplateDropDownReceiver.timers.get(notifyId);
+        String action = intent.getAction();
+        if(action.equals("PAUSE")){
+            Toast.makeText(context,"Paused", Toast.LENGTH_SHORT).show();
```

Comment:
I think the toast should probably include the timer name: something like "Timer 1 paused" also same comment for line 32